src/reconcile/rec_write.c

```diff
@@ -2533,10 +2533,19 @@ __rec_split_write(WT_SESSION_IMPL *session, WT_RECONCILE *r, WT_REC_CHUNK *chunk
         /* Turn off compression adjustment for delta. */
         compressed_size = 0;
 
-        if (F_ISSET(r->ref, WT_REF_FLAG_INTERNAL))
+        if (F_ISSET(r->ref, WT_REF_FLAG_INTERNAL)) {
             WT_STAT_CONN_DSRC_INCR(session, rec_page_delta_internal);
-        else if (F_ISSET(r->ref, WT_REF_FLAG_LEAF))
+            if (block_meta->delta_count >
+              __wt_atomic_load64(&conn->disaggregated_storage.max_internal_delta_count))
+                __wt_atomic_store64(
+                  &conn->disaggregated_storage.max_internal_delta_count, block_meta->delta_count);
```

Comment:
Same here. You should use multi->block_meta

Reply:
Updated [95e7233](https://github.com/wiredtiger/wiredtiger/pull/11779/commits/95e72339acfa73391c82a54e41372cd3493c0de2)